lib/diagnostic.rb

```diff
@@ -89,7 +111,12 @@ def example
     Example.first
   end
 
-at  describe 'DELETE destroy' do
-    # your test(s) here
+  describe 'DELETE destroy' do
+    before(:each) do
+      delete :destroy, id: examples.id, format: :json
+    end
+  it 'is succesful and returns empty array'
+  expect(resonse_status).to eq(204)
+  expect(response.body).to be_empty
   end
```

Comment:
Again, just formatting things to watch out for. 

Your `id` sent should be `example.id` and you'd check `response.status` vs `response_status`.